Applications/LogisticRegression/README.md

```diff
@@ -1,2 +1,71 @@
 # Logistic Regression
 
+This is simple logistic regression example. It has two features and 100 data in ```res/dataset1.txt```.
+
+10 test data is in ```test.txt``` for testing with inference option.
+
+Sample ```test.txt``` is like below.
+
+
+```
+2.4443 1.5438 1.0
+7.6316 4.602 1.0
+4.165 1.5636 1.0
+4.8735 2.6093 1.0
+5.5061 2.9052 1.0
+1.9383 3.6549 0.0
+5.9615 6.4565 0.0
+5.1012 7.6009 0.0
+2.8745 6.0817 0.0
+1.7358 5.4503 0.0
+```
+
+
+This exmaple uses the ```DataBufferFromCallback``` to get the training data and configuration model is in ```res/LogisticRegression.ini```
+
+```
+# Network Section : Network
```

Comment:
Could you `s/Network/Model` here and for the given ini file?

Reply:
fixed.